test/accountSpec.js

```diff
@@ -106,27 +110,38 @@ describe('Accounts', function () {
         .end()
     })
 
-    it('should return 401 when not authenticated', function *() {
+    it('should return 200/404 when not authenticated, with partial data', function *() {
```

Comment:
Maybe this could be two tests? One for the existing account and one for the nonexistent one?